### PR TITLE
Issue #SB-23174 fix: Tag objects with org and target frameworks issue fixes

### PR DESCRIPTION
**Pull request by vaibhavbhuva**

### CODE REVIEW

File: projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ export class EditorBaseComponent implements OnInit, OnDestroy, AfterViewInit {
         break;
       case 'onFormStatusChange':
         const selectedNode = this.treeService.getActiveNode();
-        if (selectedNode.data.root) { this.submitFormStatus = event.event.isValid; }
+        if (selectedNode && selectedNode.data.root) { this.submitFormStatus = event.event.isValid; }
```
Comment (venkateshwarans) [inline projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts:141]: `selectedNode.data` will always be there?

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts [hunk 2]
```diff
@@ -171,13 +173,13 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
...
-          return ["before", "after"];
-*/
-           return true;
+          /*           if(node.parent !== data.otherNode.parent){
+                      return false;
+                    }
+                    // Don't allow dropping *over* a node (would create a child)
+                    return ["before", "after"];
+          */
+          return true;
```
Comment (venkateshwarans) [inline projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:182]: remove these commented code if not useful.

**Review by venkateshwarans — COMMENTED** [review]
check comments

File: projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts (hunk 1 above)
Comment (vaibhavbhuva) [reply projects/collection-editor-library/src/lib/components/editor-base/editor-base.component.ts:141]: Yes

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts (hunk 2 above)
Comment (vaibhavbhuva) [reply projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:182]: Ok, I Will check and remove if not useful

File: projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts [hunk 3]
```diff
@@ -158,7 +160,7 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
           /** This function MUST be defined to enable dragging for the tree.
            *  Return false to cancel dragging of node.
            */
-          const draggable = _.get(this.config, 'mode') === 'Edit' ? true : false;
+          const draggable = _.get(this.config, 'mode') === 'edit' ? true : false;
```
Comment (pallakartheekreddy) [inline projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:163]: @vaibhavbhuva convert mode two lowercase and check

Comment (vaibhavbhuva) [reply projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts:164]: Fixed.

**Review by pallakartheekreddy — APPROVED** [review]

**Pull request merged**